web/src/components/IconWithUiDirection.tsx

```diff
@@ -0,0 +1,12 @@
+import styled, { css } from 'styled-components'
+
+import { UiDirectionType } from 'translations/src'
+
+export const IconWithUiDirection = styled.img<{ direction?: UiDirectionType }>`
+  ${props =>
+    props.direction === 'rtl' &&
+    css`
```

Comment:
Any reason why we need this helper?

Reply:
what do you mean with "helpers"?

Reply:
I meant the `css` helper. Why do we need that one instead of just using a regular string?